### ISSUE TITLE
plaidml.keras.backend not learning while TensorFlow does

### ISSUE THREAD

**Issue by FrankBau** [plaidml/plaidml#1753]
Followed quick start instructions on Ubuntu 18.04
Self-test looks good:
```
(plaidml) frank@Mothership17Buntu:~/PlaidML/minivgg$ plaidbench keras mobilenet
Running 1024 examples with mobilenet, batch size 1, on backend plaid
INFO:plaidml:Opening device "opencl_nvidia_geforce_rtx_3070.0"
Compiling network... Warming up... Running...
Example finished, elapsed: 0.337s (compile), 3.816s (execution)

-----------------------------------------------------------------------------------------
Network Name         Inference Latency         Time / FPS          
-----------------------------------------------------------------------------------------
mobilenet            3.73 ms                   1.20 ms / 830.25 fps
Correctness: PASS, max_error: 7.026663752185414e-06, max_abs_error: 4.172325134277344e-07, fail_ratio: 0.0
```

Tried a keras miniVGG example from here: https://www.pyimagesearch.com/2018/10/08/keras-vs-tensorflow-which-one-is-better-and-which-one-should-i-learn/

But the accuracy keeps sitting at 0.1, same like random-guess on 10 classes classification :-(

No such issues when using default tensorflow backend.

other issues mentioned BatchNormalization, yes the network is using that.

Any help highly appreciated, no idea what I could do.

Frank

**Comment by FrankBau** [comment]
after commenting out all BatchNormalizations, learning makes progress as expected.
